newIDE/app/src/Export/Builds/BuildCard.js

```diff
@@ -283,7 +293,14 @@ export const BuildCard = ({
                         if (shouldValidate(event) && nameInput.current)
                           nameInput.current.blur();
                       }}
-                      maxLength={50}
+                      onKeyDown={event => {
```

Comment:
should you use the onKeyPress just above to be consistent?

Reply:
No because the event that Dialog uses to close itself i onkeydown

Reply:
oh right